cinnabar/measurements.py

```diff
@@ -53,19 +55,51 @@ class Measurement(DefaultModel):
 
     @classmethod
     def from_experiment(cls,
-                        pIC50: float,
+                        Ki: unit.Quantity,
                         label: str,
+                        uncertainty: unit.Quantity = 0 * unit.nanomolar,
                         source: str = '',
+                        temperature: unit.Quantity = 298.15 * unit.kelvin,
                         ):
         """Create Measurement from experimental data
 
         Parameters
         ----------
+        Ki: unit.Quantity
+          experimental Ki value
+          ex.: 500 * unit.nanomolar OR 0.5 * unit.micromolar
+        label: str, optional
+          label for this data point.
+        uncertainty: unit.Quantity
+          uncertainty of the experimental value, default is zero if no uncertainty is provided (0 * unit.nanomolar)
+        source: str, optional
+          source of experimental measurement
+        temperature: unit.Quantity
+          temperature in K at which the experimental measurement was carried out. By default: 298 K (298.15 * unit.kelvin)
         """
+        R = 1.9872042586408 * 0.001 * unit.kilocalorie_per_mole / unit.kelvin # Gas constant in kcal/mol/K
+        # Convert Ki and uncertainty to molar, then strip the unit off for the math
+        Ki = Ki.to(unit.molar).m
+        uncertainty = uncertainty.to(unit.molar).m
+        if Ki > 0:
+            # dG = RT ln Ki
+            DG = R.m * temperature.m * math.log(Ki) * unit.kilocalorie_per_mole
+        else:
+            raise ValueError(
```

Comment:
I think it's the final thing. Could you add two tests to check the ValueErrors being raised here and on line 96?

Reply:
Added tests for the two ValueErrors.